src/products/iOS_products/review_iOSstrings.md

```diff
@@ -73,10 +71,12 @@ https://github.com/mozilla-mobile/firefox-ios/blame/978bf46bb680291c61c5d21b6dc2
 On the left, you should (almost always) find the bug reference:
 https://bugzilla.mozilla.org/show_bug.cgi?id=1312842
 
+Note that we now usually use GitHub issues instead of Bugzilla bugs, so `Blame` will most surely bring you to a GitHub issue.
+
 ## Wrapping up your work
 
-So you’ve reviewed the strings during the cycle, and the PR looks good? Wait! Don’t merge it quite yet.
+So you’ve reviewed the strings during the cycle, and the PR looks good - this means you can now merge the PR so the strings get exposed in Pontoon.
 
-You will need to announce this on [dev.l10n mailing list](https://lists.mozilla.org/listinfo/dev-l10n) as soon as you merge the PR. Prepare your email in advance. Here’s [an example](https://groups.google.com/forum/#!searchin/mozilla.dev.l10n/ios$20v7%7Csort:relevance/mozilla.dev.l10n/oR5u3MdMLgE/952I4eyADAAJ) of what it should contain.
+Pontoon ([here for Firefox for iOS](https://pontoon.mozilla.org/projects/firefox-for-ios/) and [here for Focus for iOS](https://pontoon.mozilla.org/projects/focus-for-ios/) should get the new strings automatically after a few minutes, but it’s always a good idea to double check just in case.
```

Comment:
```suggestion
Pontoon ([here for Firefox for iOS](https://pontoon.mozilla.org/projects/firefox-for-ios/) and [here for Focus for iOS](https://pontoon.mozilla.org/projects/focus-for-ios/)) should get the new strings automatically after a few minutes, but it’s always a good idea to double check just in case.
```